.github/workflows/update_term_vocab_files.yaml

```diff
@@ -104,8 +119,9 @@ jobs:
 
       - name: Commit and push if any community config files have changed
         run: |
-          if ! git diff --quiet configs; then
-            git add configs
+          git status
+          git add configs
+          if ! git diff --cached --quiet configs; then
```

Comment:
Could we add an echo statement here (or make the git diff not silent) so we can more easily see if the diff is empty when we don't expect it to be